### PR TITLE
Update Regresion Tests to account for Floating Point Error

### PR DESCRIPTION
**Pull request by JohnEEnglish**
Also updated some text instructions since we are now using 'setup.bash' not 'setup.sh'

### CODE REVIEW

File: regression-tests/convert-FITS_+_generate_fit_quality/convert-FITS_+_generate_fit_quality.sh [hunk 1]
```diff
@@ -2,20 +2,27 @@
...
 TEST_RESULT=1
 
-diff <(cut -f1-39,42-150 -d$','  $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.out/galaxy.csv) <(cut -f1-39,42-150 -d$','  $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/test.out/galaxy.csv) > comp.txt
+sed 's/[^0-9.EefF-]/ /g' $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.out/galaxy.csv > /tmp/sPaRCfIrE_TC1
+sed 's/[^0-9.EefF-]/ /g' $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/test.out/galaxy.csv > /tmp/sPaRCfIrE_TC2
 
-if ! [ -s "comp.txt" ]; then
+paste /tmp/sPaRCfIrE_TC1 /tmp/sPaRCfIrE_TC2 > /tmp/sPaRCfIrE_TCP1
+
+awk '{for(i=1;i<=NF;i++) if(abs($1 $2) > 1.0e-6) print "different"}' > /tmp/sPaRCfIrE_TCP1
```
Comment (waynebhayes) [inline regression-tests/convert-FITS_+_generate_fit_quality/convert-FITS_+_generate_fit_quality.sh:19]: On what system have you ever seen "abs" as a built-in function to awk? This won't work; plus you've put ($1 $2) without a minus sign between them.

To make it work, it's easy: add the following before the rest of the awk code:

function abs(a,b) { if(a>b) return a-b; else return b-a;}

File: regression-tests/convert-FITS_no_arguments/convert-FITS_no_argument.sh [hunk 2]
```diff
@@ -3,19 +3,28 @@
 die() { echo "FATAL ERROR IN CONVERT FITS: $@" >&2; exit 1;}
 TEST_RESULT=1
 
+abs(){
```
Comment (waynebhayes) [inline regression-tests/convert-FITS_no_arguments/convert-FITS_no_argument.sh:6]: Oh. Um. This is a Bourne shell function, and won't affect anything inside awk. Have you actually tried running this regression test?

**Review by waynebhayes — CHANGES_REQUESTED** [review]
Does this actually run? I can't imagine it works; you need to define an abs() function *inside awk*, not in the Bourne shell. See my comment inside for details.

**Comment by JohnEEnglish** [comment]
Sorry my internet is out, I'll make the change  When it comes back.

**Pull request closed without merging**